tests/Unit/Evolution/DiscontinuousGalerkin/Test_BoundaryCorrectionsHelper.cpp

```diff
@@ -52,8 +81,8 @@ struct System {
   using compute_volume_time_derivative_terms = TimeDerivativeTerms;
 };
 
-template <size_t Dim>
-struct Correction {
+template <size_t Dim, typename VolumeDoubleType>
+struct Correction final {
```

Comment:
This `Correction` object is just for the test helper, but presumably its interface will be the same as the forthcoming individual boundary correction structs -- will the boundary corrections need to store member data? If not, can their functions be made `static`?

Reply:
Some boundary corrections (not ones we are using) exist that have parameters that would be reasonable to tune in an input file. I think it's easier to have all the functions be non-`static` for uniformity and ease-of-understanding for new users. There are also classes of boundary corrections like HLL, HLLE, HLLEM, HLLC, etc. (there's something like 5-8 of these that I know of, but I'm sure there are even more) that could reasonably all be one class with an input file option for which one to use. This helps keep down the amount of boiler plate code (class, inheritence, function declarations, files, etc.) to support many different forms of nearly identical boundary corrects.